cfr-fxa.yaml

```diff
@@ -17,54 +17,9 @@
       string_id: cfr-doorhanger-bookmark-fxa-header
   frequency:
     lifetime: 3
-  targeting: locale != 'ru' && !usesFirefoxSync && isFxAEnabled == true && totalBookmarksCount
+  targeting: !usesFirefoxSync && isFxAEnabled == true && totalBookmarksCount
```

Comment:
Dropped the `ru` restriction. This was fixed in https://bugzilla.mozilla.org/show_bug.cgi?id=1548784.